api/ensure-zoom-authenticated.js

```diff
@@ -1,9 +1,20 @@
+import crypto from "crypto";
 // this is a helper method to make sure the zoom request we get is authentic
 export default async (req, res, callback) => {
-  const secret = process.env.ZOOM_VERIFICATION_TOKEN
-  if (!secret || req.header('authorization') == secret) {
-    await callback()
+  if (isZoomAuthenticRequest(req)) {
+    await callback();
   } else {
-    res.status(403).send('Missing/invalid Zoom verification token')
+    res.status(403).send('Unauthorized sender');
```

Comment:
Can we also please add a new counter metric for this scenario?  We already have a precedent for these generic errors: `errors.<keyname>`